app/components/schedule/Home.vue

```diff
@@ -95,8 +95,9 @@ export default {
       this.$axios.get(this.streamServer).then((response) => {
         this.nowPlaying = response.data
       }).catch((error) => {
-        console.log(error)
-        this.$nuxt.error({ statusCode: 500, message: 'Schedule not available' })
+        console.log('Network interrupted stream. Automatically reconnecting shortly...', error)
+        this.$sentry.captureException(new Error('Stream interrupted ', error))
+        this.interval = setTimeout(this.checkNowPlaying, 15000)
```

Comment:
If I understand this correctly, if the nowPlayingData request fails, we check again only once after 15 seconds. Is this the expected functionality? Should we reset the interval timer if the 15 second check results successfully?

Reply:
@molnar-a Thanks, I made a clear on succesfull response